.github/dependabot.yml

```diff
@@ -9,10 +9,14 @@ updates:
     allow:
       - dependency-type: "all"
     assignees:
-      - "dotnet/aspnet-build"
+      - "dougbu"
+      - "juntaoluo"
+      - "wtgodbe"
+    commit-message:
+      prefix: "[main] "
+      include: scope
```

Comment:
I'm not positive what `include: scope` does because the description of the default configuration w/o a `commit-message` setting is pretty vague (something about the conventions observed in our repo). I hope this means "keep the default information seen at the moment in https://github.com/dotnet/aspnetcore/pulls/app/dependabot". If it adds even more information, I'm fine as long as the PRs don't get uselessly noisy.